includes/core/classes/class-setup.php

```diff
@@ -417,7 +417,7 @@ public function check_users_can_register(): void {
 			update_option( 'gatherpress_suppress_membership_notification', true );
 		} else {
 			Utility::render_template(
-				sprintf( '%s/includes/templates/admin/setup/membership-check.php', GATHERPRESS_CORE_PATH ),
+				sprintf( '%s/includes/templates/admin/setup/site-check.php', GATHERPRESS_CORE_PATH ),
```

Comment:
Can you search/replace `gatherpress_suppress_membership_notification` and change to `gatherpress_suppress_site_notification` too? Thx!